### DISCUSSION TITLE
More data fields

### DISCUSSION THREAD

**Discussion by pinkysworld** [powerpaul17/nc_money#13] [Ideas]
Hi,
I would love to see more fields for importing CSV - my bank for example is using the following data fields in the CSV (translated to English):

Posting date, Value date, Transaction type, Posting text, Amount, Currency, ordering party account, Bank code ordering party account, IBAN ordering party account

Some data visualization feature would be great, diagrams, charts, etc :)

**Comment by powerpaul17** [comment]
Hey, I know that it would be great to store all information from your bank in the transactions, I just haven't figured out yet what's the best way to do it. I am a bit hesitant to add fields to the database tables as this is the hardest thing to modify later. Maybe we could store additional information in a field which contains then all remaining data from the import.

Data visualization is of course planned and worked on.. :wink:
